node/pkg/guardiansigner/guardiansigner_test.go

```diff
@@ -0,0 +1,88 @@
+package guardiansigner
+
+import (
+	"testing"
+
+	"github.com/ethereum/go-ethereum/crypto"
+	ethcrypto "github.com/ethereum/go-ethereum/crypto"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParseSignerUri(t *testing.T) {
+	tests := []struct {
+		label        string
+		path         string
+		expectedType SignerType
+	}{
+		{label: "RandomText", path: "RandomText", expectedType: InvalidSignerType},
+		{label: "ArbitraryUri", path: "arb://data", expectedType: InvalidSignerType},
+		// File
+		{label: "FileURI", path: "file://whatever", expectedType: FileSignerType},
+		{label: "FileUriNoDotDotSlash", path: "filewhatever", expectedType: InvalidSignerType},
+		{label: "FileUriMultipleDotDotSlash", path: "file://testing://this://", expectedType: FileSignerType},
```

Comment:
There are no `DotDotSlash` sequences in either of these. Maybe you meant scheme separators `://`?

Reply:
Updated the naming.